lib/fuschia/entities/pesquisador.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Fuschia.Entities.Pesquisador do
+  @moduledoc """
+  Pesquisador Schema
+  """
+  use Fuschia.Schema
+  import Ecto.Changeset
+
+  alias Fuschia.Entities.Pesquisador
+  alias Fuschia.Entities.Universidade
+  alias Fuschia.Entities.User
+  alias Fuschia.Types.TrimmedString
+
+  @required_fields ~w(cpf_usuario minibibliografia tipo_bolsa link_lattes universidade_id)a
+  @optional_fields ~w(cod_orientador)a
+
+  @primary_key {:cpf_usuario, TrimmedString, []}
+  schema "pesquisador" do
+    field :minibibliografia, :string
+    field :tipo_bolsa, :string
```

Comment:
o `tipo_bolsa` vai ser um `Ecto.Enum`, assim como `perfil` do schema `User`
```suggestion
   field :tipo_bolsa, Ecto.Enum, values: @tipos_bolsa, default: "<default>"
```